corporate/views/billing_page.py

```diff
@@ -37,6 +37,16 @@
 
 billing_logger = logging.getLogger("corporate.stripe")
 
+CARD_CAPITALIZATION = {
+    "amex": "American Express",
+    "diners": "Diners Club",
+    "discover": "Discover",
+    "jcb": "JCB",
+    "mastercard": "MasterCard",
+    "unionpay": "UnionPay",
+    "visa": "Visa",
+}
```

Comment:
Comparing with https://stripe.com/docs/payments/cards/supported-card-brands, I think `Mastercard` is the correct spelling; and we might want to use other forms of some of the others?

Reply:
Confirmed only that change seems helpful.